src/init.ts

```diff
@@ -143,29 +143,30 @@ export default function init(undefinedSymbol: symbol, foreignCallableHooksCallba
         return r;
     }
 
+
     function createShadowTarget(
-        typeofTarget: string,
-        protoInTarget: boolean | undefined,
-        functionNameOfTarget: string | undefined,
-        isTargetAnArray: boolean | undefined,
+        targetTypeof: string,
+        targetIsArrowFunction: boolean | undefined,
```

Comment:
flipping the value of this argument to make it easier and match other arguments.